### DISCUSSION TITLE
How to navigate to different Folder Page in Nextjs

### DISCUSSION THREAD

**Discussion by asenseofpradhyu** [vercel/next.js#25337] [Help]
Project structure is like

/pages
  --/user
    ----login.js
    ----register.js
  --/admin
    ----dashboard.js

What i want to do is when user successfully login(login url:-> http://localhost:3100/user/login) then user should navigate to dashboard page(Expected url:->http://localhost:3100/admin/dashboard). Anyone help me with this please..

**Comment by scottschindler** [comment]
Hey!

You can use the Next.js built-in router. (https://nextjs.org/docs/api-reference/next/router).

First, you need to import router:

`import { useRouter } from "next/router";`

Second, you need to define router inside the component:

`const router = useRouter();`

Last, after a successful log-in, at the end of that function, you can write something like: 

` router.push("/admin/dashboard");`

Hope this helps. Definitely check out the docs for more info!

**Reply by asenseofpradhyu** [reply]
@scottschindler Thanks 👍
